app/views/partials/theme/simple/items/show.liquid

```diff
@@ -1,3 +1,18 @@
+{% liquid
+  assign product_rating = 0
+  assign product_point = 0
+  if item.reviews.size 
+    for review in item.reviews
```

Comment:
you can take a look at liquid, it allows `else` statement for `for`, which will be invoked if collection is empty - https://documentation.platformos.com/api-reference/liquid/loops#handling-an-empty-collection